src/toil/batchSystems/abstractGridEngineBatchSystem.py

```diff
@@ -382,9 +377,14 @@ def getWaitDuration(self):
     def getRescueBatchJobFrequency(cls):
         return 30 * 60 # Half an hour
 
-    @classmethod
-    def sleepSeconds(cls):
-        return 1
+    def sleepSeconds(cls, sleeptime=None):
```

Comment:
Shouldn't this signature be `def sleepSeconds(self, sleeptime=None):`?
(`self`, not `cls`)